LocalSettings.php

```diff
@@ -4876,10 +4877,10 @@
 		'default' => false,
 	],
 	'wgWikiSeoEnableAutoDescription' => [
-		'default' => false,
+		'default' => true,
```

Comment:
overridedefault needs changed in ManageWikiSettings.php.

Reply:
Should this be changed back to 'false' then?

Reply:
No, it should match ManageWikiSettings.php

Reply:
Changed then, thanks.